### DISCUSSION TITLE
How do I use this?

### DISCUSSION THREAD

**Discussion by MoeMonsuta** [OwO-Network/DLX#67] [General]
When I load the program it just says "DeepL X has been successfully launched!" and sits there doing nothing. How do I insert text to translate?

**Comment by missuo** [comment]
You need to request the API

**Reply by YukinXX** [reply]
How can I get one? I don't understand.

**Reply by burger2077** [reply]
**Does this api refer to deepl's api?**

**Comment by AnilRana675** [comment]
can any one tell how can i run on Windows??
